### PR TITLE
shitty fix to operatives on lavaland not having access to their own base + some consistency

### PR DESCRIPTION
**Pull request by ghost**
1. i gave the scientists agent IDs since I can't figure out why the syndicate access doors aren't responding to syndicate-access IDs and i'm not sure what kor did with the access things there it makes my head fucking hurt

2. i gave the scientists the eswords and the comms agent the sniper because what the fuck, why do the SCIENTISTS have more heavy weapons than the comms guy

3. comms guy has chameleon shit too just for uh, flavor i guess

:cl: NOT SYNDICATES
add: Lavaland syndicate operatives have been given agent IDs so they can access their own fucking base
fix: fixed lavaland ops not having fucking access to their base
tweak: Lavaland operative scientists have eswords, lavaland operative comm agents have chameleon gear and the sniper instead of vice versa
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
I cannot support this if it results in fewer rifles (three scientists, one comms guy). Find a better alternative, please.

**Comment by ExcessiveUseOfCobblestone** [comment]
what's wrong with the ID?

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -130,6 +130,7 @@
 	radio = /obj/item/device/radio/headset/syndicate/alt
 	back = /obj/item/weapon/storage/backpack
 	pocket1 = /obj/item/weapon/gun/ballistic/automatic/pistol
+	pocket2 = /obj/item/weapon/card/id/syndicate/anyone
 	id_job = "Operative"
 	id_access = "Syndicate"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/ruins/lavaland_ruin_code.dm]: lowercase this and see if that fixes the scientist problem.

**Comment by calzilla1** [comment]
Rifle for comms guy, real c20 (maybe a bio sprayer?)for scientists

**Comment by Armhulen** [comment]
i like the bio sprayer idea

**Comment by KorPhaeron** [comment]
cobby posted the actual fix, gonna just do that

**Pull request closed without merging**